### DISCUSSION TITLE
Resolving library dependencies across evaluation tools and models

### DISCUSSION THREAD

**Discussion by olachinkei** [UpstageAI/evalverse#22] [Q&A] [answered]
Thank you for the amazing tool. It seems that the required libraries are specified at https://github.com/UpstageAI/evalverse/blob/main/pyproject.toml. However, what should we do if there is a version conflict with any of the libraries in the future?

Also, I believe that the libraries required may vary from model to model. In such cases, what would be the best practice to follow?

**Comment by jihoo-kim** [comment, marked as answer]
That's a really good comment.

We plan to upgrade the version of evalverse by ensuring that there are no conflicting versions of the required libraries.
For the inevitable conflicts you mentioned, we would like to introduce [a method to manage multiple versions](https://docs.metaflow.org/scaling/dependencies/libraries) to resolve version conflicts.

**Reply by olachinkei** [reply]
@jihoo-kim , Thank you very much for your answer! I think how to establish an environment without version confliction with metaflow!
